apps/01PigLatin.js

```diff
@@ -3,49 +3,92 @@
 var assert = require('assert');
 var readline = require('readline');
 var rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
+    input: process.stdin,
+    output: process.stdout
 });
 
 
 function pigLatin(word) {
 
-  // Your code here
+    //  var word = 'car';
+    var firstLetter = word[0];
+    console.log(firstLetter);
+    word = word.toLowerCase();
 
+    var word = 'crazy';
```

Comment:
You have an argument named `word` being passed into the function, but then you're also setting a new variable here that's also called `word`. Can you think of how that might be problematic?

Recall that an argument being passed into a function is named arbitrarily - it does not need to be declared as a variable; it simply represents value - `String`, `Number`, even a `function()` - being passed into the function.